SayTheirNames.xcodeproj/project.pbxproj

```diff
@@ -1680,10 +1683,11 @@
 			baseConfigurationReference = 72373A43202B7215526324F7 /* Pods-SayTheirNames.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CODE_SIGN_ENTITLEMENTS = "SayTheirNames/Say Their Names.entitlements";
 				CODE_SIGN_IDENTITY = "iPhone Distribution: Franck Ndame (Q93T3ZJQ58)";
 				CODE_SIGN_STYLE = Manual;
-				CURRENT_PROJECT_VERSION = 2;
-				DEVELOPMENT_TEAM = "";
+				CURRENT_PROJECT_VERSION = 3;
+				DEVELOPMENT_TEAM = Q93T3ZJQ58;
```

Comment:
```suggestion
				DEVELOPMENT_TEAM = "";
```
let's revert it back. (we worked on this together so I will just commit the suggestion right away)